web/webpack.config.js

```diff
@@ -92,10 +93,11 @@ const config = {
       }
     ]
   },
-  // replaces "process.env.____" with the values defined in the actual
-  // environment at build time
   plugins: [
-    new MiniCssExtractPlugin(),
+    new MiniCssExtractPlugin({ filename: '[name].[contenthash].css' }),
```

Comment:
This plugin now supports hashing itself, which means our CSS files can have hashes in their filenames just like the JS files do.